tcms/testruns/models.py

```diff
@@ -143,7 +143,8 @@ def create_execution(
             case_text_version=case_text_version,
             build=build or self.build,
             sortkey=sortkey,
-            close_date=None,
+            stop_date=None,
+            start_date=None,
         )
```

Comment:
you are changing the signature method here which means you have to update all invocation of this method as well.